### PR TITLE
Update whats-new-in-office-365-atp.md

### PR DESCRIPTION
**Pull request by johnengels**
Comments for the end of 2020 improvements for AIR

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/security/office-365-security/whats-new-in-office-365-atp.md [hunk 1]
```diff
@@ -41,6 +41,7 @@ This article lists new features in the latest release of Microsoft Defender for
 ## December 2020
 
 - [Secure by default in Office 365](secure-by-default.md)
+- Automated investigation improvements to:  general alerts for manually triggered email investigations, treat mailbox changes as a separate entity category, remove redundant URL block actions, and create outbound email clusters for user compromise invesetigations.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/whats-new-in-office-365-atp.md]:
```suggestion
- Automated investigation improvements to: general alerts for manually triggered email investigations, treat mailbox changes as a separate entity category, remove redundant URL block actions, and create outbound email clusters for user compromised investigations.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@MSFTTracyP Please merge this PR. Thanks.

**Pull request merged**